### DISCUSSION TITLE
Screens/Barriers implementation

### DISCUSSION THREAD

**Discussion by ioannides-dB** [Universite-Gustave-Eiffel/NoiseModelling#299] [Q&A] [answered]
Hi, I have recently discovered your software during an initial reasearch for my masters thesis on the use of Cnossos method. Nice work! Impressed by the open source possibilities in general as I am relatively new to this. 

I intend to do a case study of an urban area and propose noise mitigation which will include barriers/screens. I have used the software to produce some baseline results. Id now like to introduce screens. I see you have a "screens to buildings" block. Is there a screens layer that I am not aware of?  You have some documantation on that? Or how to implement screens in general? 

Thanxs a lot.
Stefanos

**Comment by pierromond** [comment, marked as answer]
Dear stefanos, 

Thank you for your nice message. In fact screens to buildings allows to join a screen table (LINESTRING type) to a buildings table (POLYGON type). If you want, you can also create your screens directly in the BUILDINGS table saying that they are very thin buildings. It will work the same way!

Good luck!
